src/assets/css/style.scss

```diff
@@ -0,0 +1,33 @@
+#app {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  text-align: center;
+  color: #2c3e50;
+}
+
+#nav {
+  padding: 30px;
+
+  a {
+    font-weight: bold;
+    color: #2c3e50;
+
+    &.router-link-exact-active {
+      color: #42b983;
+    }
+  }
+}
+
+body {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
```

Comment:
May be need to remove one line 
`font-family: Avenir, Helvetica, Arial, sans-serif;`
or
`font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;`